test-fixtures/engine-config/engine.config.js

```diff
@@ -1,6 +1,3 @@
-/**
```

Comment:
why remove comment?

Reply:
I can restore it. I thought it might cause the linting issues I had with the branch